tensorflow/compat_template_v1.__init__.py

```diff
@@ -45,13 +45,6 @@
   _current_module.__path__ = [_module_dir] + _current_module.__path__
 setattr(_current_module, "keras", keras)
 
-# Explicitly import lazy-loaded modules to support autocompletion.
-# pylint: disable=g-import-not-at-top
-if _typing.TYPE_CHECKING:
-  from tensorflow_estimator.python.estimator.api._v1 import estimator
-# pylint: enable=g-import-not-at-top
-
-
 from tensorflow.python.platform import flags  # pylint: disable=g-import-not-at-top
```

Comment:
Not sure why the build is broken. maybe there is an issue with this statement above `# pylint: disable=g-import-not-at-top`  because now I'm re-enabling it at the bottom of the file, line 84